plugins/ReactNativeStyle/ReactNativeStyle.js

```diff
@@ -102,6 +102,10 @@ class NativeStyler extends React.Component {
   _handleStyleRename(oldName: string, newName: string, val: string | number) {
     var style = shallowClone(this.state.style);
     delete style[oldName];
+    if (newName === '') {
+      this.setState({style});
+      return;
```

Comment:
Hi, @jhen0409. Thanks for your comment. Can you please help me with build process for backend code? I tried to do some changes in `plugins/ReactNativeStyle/setupBackend.js`, then run `npm run backend` inside `packages/react-devtools-core`, then run `npm start` inside `packages/react-devtools` directory but it seems like my changes doesn't have any affect on react devtools and it uses old/existing code. I'm really new to this project so it looks like I've missed something important :)